examples/swift/greeter.swift

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env xcrun -sdk macosx swift
+
+import Foundation
+
+while(true){
```

Comment:
The magic is in `.availableData`. This method reads the input buffer or blocks if no data is available. Without it, the loop would run non-stop :)

So it's more or less equivalent to `stdin = gets` in ruby.